code/modules/mob/inventory.dm

```diff
@@ -240,9 +240,10 @@
 	if(del_on_fail)
 		qdel(I)
 		return FALSE
-	I.forceMove(drop_location())
+	var/atom/location = drop_location()
```

Comment:
This cast make sense if you're trying to reduce proc calls

Reply:
yeet